manifests/metrics-server/metrics-server-deployment.yaml

```diff
@@ -39,10 +39,10 @@ spec:
         emptyDir: {}
       containers:
       - name: metrics-server
-        image: %{SYSTEM_DEFAULT_REGISTRY}%rancher/mirrored-metrics-server:v0.5.2
+        image: %{SYSTEM_DEFAULT_REGISTRY}%rancher/mirrored-metrics-server:v0.6.1
         args:
         - --cert-dir=/tmp
-        - --secure-port=4443
+        - --secure-port=10250
```

Comment:
Why did the secure port get changed?

Reply:
* https://github.com/kubernetes-sigs/metrics-server/pull/1054